cfgov/v1/wagtail_hooks.py

```diff
@@ -9,6 +9,7 @@
 @hooks.register('after_create_page')
 @hooks.register('after_edit_page')
 def share_the_page(request, page):
```

Comment:
The Page object has a property method for this called `specific` http://docs.wagtail.io/en/v1.2/reference/pages/model_reference.html#wagtail.wagtailcore.models.Page.specific